site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -138,14 +133,43 @@ Options are:
    For now, use a prefix in the endpoint configuration to limit the scope of each capture to data of a single file type.
    :::
 
-Only CSV data requires further configuration. When capturing CSV data, you must specify:
+#### CSV configuration
+
+Only CSV data allows further configuration. When capturing CSV data, you may additionally specify:
```

Comment:
I think it's worth mentioning that the default is to automatically determine the values for delimiter, escape char, quote char, and encoding on a best-effort basis. Most users won't need to provide any configuration for parsing CSVs unless they're doing something weird.